ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -52,6 +52,14 @@
   : $doc.display($propName)
   #end
 #end
+#macro (setPollTitle)
+  #set($meetingName = $doc.name)
+  #set($defaultDocName = $services.model.getEntityReference('DOCUMENT', 'default').name)
+  #if($meetingName == $defaultDocName)
+    #set($meetingName = $doc.documentReference.lastSpaceReference.name)
+  #end
+  #set($pollName="XPoll for the Meeting $meetingName")
```

Comment:
Well, if you call this `$pollName`, then the macro should set the poll name and not the poll title, right?

Reply:
Changing it right now.